NanoProd/crab/Run3_2022/DY_amcatnlo.yaml

```diff
@@ -30,6 +30,8 @@ DYto2L_M_50_0J_amcatnloFXFX: /DYto2L-2Jets_MLL-50_0J_TuneCP5_13p6TeV_amcatnloFXF
 DYto2L_M_50_1J_amcatnloFXFX: /DYto2L-2Jets_MLL-50_1J_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
 DYto2L_M_50_2J_amcatnloFXFX: /DYto2L-2Jets_MLL-50_2J_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
 
+DYto2Mu_MLL_105to160_amcatnloFXFX: /DYto2Mu-2Jets_MLL-105To160_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
```

Comment:
Move the exclusive DY to a separate file. Soon, there will be more coming for DYto2Tau. I propose to call it `DY_amcatnlo_excl.yaml`.